config/jupyterhub_config.py

```diff
@@ -61,6 +61,16 @@
 c.DockerSpawner.remove = environment != 'dev'
 c.DockerSpawner.debug = True
 
+# Set extra labels in order to use Rancher's network policies
+# ref: https://rancher.com/docs/rancher/v1.6/en/rancher-services/networking/#containers-created-with-the-docker-cli
+c.DockerSpawner.extra_create_kwargs = {
+    'labels': {
+        'io.rancher.container.network': 'true'
+    }
+}
```

Comment:
I assume this has no effect in docker compose?

Reply:
yea. I tested locally. docker compose works fine.